Info about settings variable. (#4493)
segmentio/segment-docs: src/connections/functions/destination-functions.md

When you deploy your destination function in your workspace, you fill out the settings on the destination configuration page, similar to how you would configure a normal destination.

> info ""

> > The settings object can't be accessed as a global variable in the code. It has to be passed to the function at runtime.

Reviewer: ```suggestion
> You must pass the settings object to the function at runtime. Functions can't access the settings object when it's stored as a global variable.
```
